optimize/components/userguide/additional-features/alerts.md

```diff
@@ -4,6 +4,10 @@ title: Alerts
 description: Get a notification as soon as your system is behaving in an unexpected manner.
 ---
 
+:::note
+Alerting with webhooks is only available for Camunda 7.
+:::
```

Comment:
Just wondering if this note would be better placed under the `## Send alerts to external systems` section. I would be concerned that people might see this and overlook the whole alerting section, assuming its unavailable in C8, which isn't quite correct

Reply:
Great point, we should move it. I'll adjust this admonition for each of the versions to be within that section.